src/codegen/persistence.ml

```diff
@@ -0,0 +1,283 @@
+(* Support for orthogonal persistence using stable heap and in-place memory upgrades 
```

Comment:
Is this file still used? I ask because its not producing a candid type encoding (no lebs) so I suspect its old.

Reply:
Thank you. Indeed, this is unused. I have deleted it.